msamp/nn/__init__.py

```diff
@@ -9,6 +9,8 @@
 from msamp.nn.linear import FP8Linear, LinearReplacer
 from msamp.nn import functional
 from msamp.nn.clip_grad import clip_grad_norm_
+from msamp.nn import distributed
 del functional
+del distributed
```

Comment:
why this is needed?

Reply:
`from msamp.nn import distributed` will call `torch.nn.parallel.DistributedDataParallel = FP8DistributedDataParallel`, which is neccessary. However, just like msamp.nn.functional, msamp.nn.distributed is something internal in MS-AMP and should not be exposed. So just as what we did to functional, we `del distributed` after importing it.